cardano-api/src/Cardano/Api/IO/Compat.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.Api.IO.Compat
```

Comment:
Let's put this in `Cardano.Api.IO`, a module for a single function is overkill.

Reply:
Done!